dispatch/BESInterface.cc

```diff
@@ -32,18 +32,18 @@
 
 #include "config.h"
 
-#include <string>
-#include <sstream>
-#include <iostream>
+#include <cstdlib>
 
+#include <signal.h>
 #if HAVE_UNISTD_H
 #include <unistd.h>
 #endif
 
-using std::string;
-using std::ostringstream;
-using std::bad_alloc;
-using std::cout;
+#include <setjmp.h> // Used for the timeout processing
```

Comment:
should this be conditional based on if on windows system? I've seen the macros in other places but not here. Or is this a generic include?